docs/learn/expert-guides/universal-profile/register-lsp12-issued-assets.md

```diff
@@ -0,0 +1,214 @@
+---
+sidebar_label: 'Register LSP12 Issued Assets'
+sidebar_position: 5
+description: Learn how to register Issued Assets under an Unviersal Profile.
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import Details from '../../../../src/components/Details'
+
+# Set LSP8 NFT Metadata
+
+In this guide you will learn how to add a Digital Asset (either [LSP7 Digital Asset](../../../standards/tokens/LSP7-Digital-Asset.md) or [LSP8 Identifiable Digital Asset](../../../standards/tokens/LSP8-Identifiable-Digital-Asset.md)) to the LSP12 Issued Assets of a Universal Profile.
+
+## Setup
+
+You will need the address of an existing LSP7 or LSP8 Digital Asset in order to follow this tutorial. E.g:
+
+- LSP7 Digital Asset [`0xf7056bdE90f494F55967858F1e9E4AFB1026C5C8`](https://explorer.execution.testnet.lukso.network/address/0xf7056bdE90f494F55967858F1e9E4AFB1026C5C8?tab=read_contract).
+- LSP8 Digital Asset [`0xf651b88925C0B6C81Ad6f658a2F104226d837F60`](https://explorer.execution.testnet.lukso.network/address/0xf651b88925C0B6C81Ad6f658a2F104226d837F60?tab=read_contract).
+
+The following code snippets require the installation of the following libraries:
+
+- [`ethers.js`](https://github.com/ethers-io/ethers.js/) or [`web3.js`](https://www.npmjs.com/package/web3)
+- [`@lukso/lsp-smart-contracts`](https://github.com/lukso-network/lsp-smart-contracts/)
+- [`@erc725/erc725.js`](https://github.com/ERC725Alliance/erc725.js/)
+
+<Tabs groupId="web3-lib">
+  <TabItem value="web3js" label="web3.js">
+
+```shell
+npm install web3 @lukso/lsp-smart-contracts @erc725/erc725.js
+```
+
+  </TabItem>
+  <TabItem value="ethersjs" label="ethers.js">
+
+```shell
+npm install ethers @lukso/lsp-smart-contracts @erc725/erc725.js
+```
+
+  </TabItem>
+</Tabs>
+
+## Imports and constants
+
+Import `web3.js`/`ethers`, the [`LSP8IdentifiableDigitalAsset`](../../../contracts/contracts/LSP8IdentifiableDigitalAsset/LSP8IdentifiableDigitalAsset.md) ABI from [`@lukso/lsp-smart-contracts`](../../../contracts/introduction.md) and create an instance of this contract with the `lsp8ContractAddress`.
+
+<Tabs groupId="web3-lib">
+  <TabItem value="web3js" label="web3.js">
+
+```javascript
```

Comment:
As we want to sync our docs guides with the playground, this snippet (using a private key) should also be included there. Could you create a PR and link it here? This would also make testing easier.